dbschema/partner.esdl

```diff
@@ -0,0 +1,38 @@
+module default {
+  type Partner extending Resource, Project::ContextAware, Mixin::Pinnable, Mixin::Taggable {
+    required active: bool;
+    required globalInnovationsClient: bool;
+
+    pmcEntityCode: str {
+      constraint regexp(r'^[A-Z]{3}$');
+    }
+
+    #address: str; #TODO - this needs figured out - needed on here and Organization?
+    types: array<Partner::Type>;
+    financialReportingTypes: array<Partner::FinancialReportingType>;
+
+    pointOfContact: default::Person;
+    single languageOfWiderCommunication: default::Language;
+    required organization: default::Organization;
+    required languagesOfConsulting: default::Language;
+    required fieldRegions: default::FieldRegion;
+    required countries: default::Location;
```

Comment:
```suggestion
    required multi languagesOfConsulting: default::Language;
    required multi fieldRegions: default::FieldRegion;
    required multi countries: default::Location;
```

Reply:
I thought `multi` is implicit on links?

Reply:
_Stored_ links are `single` by default. _Backlinks_ are `multi` by default.